InlamningEtt.Tests/BankTests.cs

```diff
@@ -5,6 +5,41 @@ namespace InlamningEtt.Tests
 {
     public class BankTests
     {
+        //    Skapa en enhetstest som verifierar att det inte g�r att �verf�ra mer pengar �n det finns saldo p� fr�n-kontot.
+        [Fact]
+        public void Transfer_VerifyAccountBalanceAfterTransfer()
+        {
+            // Arrange
+            var repo = new BankRepository();
+            var fromAccount = repo.GetAccount(20);
+            var toAccount = repo.GetAccount(25);
+            fromAccount.Balance = 200M;
+            toAccount.Balance = 500M;
+            const decimal amount = 100M;
+            // Act
+            var success = repo.Transfer(fromAccount.Id, toAccount.Id, amount);
+            // Assert
+            Assert.True(success);
+            Assert.Equal(100M, fromAccount.Balance);
+            Assert.Equal(600M, toAccount.Balance);
+        }
+        [Fact]
+        public void Transfer_MoreThanBalance()
+        {
+            // Arrange
+            var repo = new BankRepository();
+            var fromAccount = repo.GetAccount(20);
+            var toAccount = repo.GetAccount(25);
+            fromAccount.Balance = 200M;
+            toAccount.Balance = 500M;
+            const decimal amount = 300M;
+            // Act
+            var success = repo.Transfer(fromAccount.Id, toAccount.Id, amount);
+
+            // Assert
+            Assert.False(success);
+            Assert.Equal(200M, fromAccount.Balance);
```

Comment:
Lägg till att du ska kontrollera att saldot på kontot överföringen går till ska vara oförändrad också.